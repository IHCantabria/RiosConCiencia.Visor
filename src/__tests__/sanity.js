import { shallowMount } from "@vue/test-utils";
import merge from "lodash.merge";

const Component = {
  render() {},
  title: "dummy title"
};
function createWrapper(overrides) {
  const defaultMountingOptions = {
    propsData: {}
  };
  return shallowMount(Component, merge(defaultMountingOptions, overrides));
}

describe("sanity test", () => {
  //UNIT TESTS
  it("unit", () => {
    expect.assertions(2);
    const wrapper = createWrapper();
    expect(2).toBe(2);
    expect(wrapper).toBeDefined();
  });
  //SNAPSHOT TESTS
  it("renders correctly", () => {
    expect.assertions(1);
    const wrapper = createWrapper();
    expect(wrapper.element).toMatchSnapshot();
  });
});
