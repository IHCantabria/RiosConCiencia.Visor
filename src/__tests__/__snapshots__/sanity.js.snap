// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sanity test renders correctly 1`] = `<!---->`;
